js/glowingbear.js

```diff
@@ -746,6 +746,31 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         }
     };
 
+    function nickFromBufferline(bufferline) {
```

Comment:
Well, in any case `nick` should be an attribute of the bufferline (and it's constructed just like like `rtext`).